ra/ra.go

```diff
@@ -733,12 +737,16 @@ func (ra *RegistrationAuthorityImpl) checkAuthorizationsCAA(
 	return nil
 }
 
-func (ra *RegistrationAuthorityImpl) recheckCAA(ctx context.Context, names []string) error {
+// recheckCAA accepts a list of of names that need to have their CAA records
+// rechecked because their associated authorizations are sufficiently old and
+// performs the CAA checks required for each. If any of the rechecks fail an
+// error is returned.
+func (ra *RegistrationAuthorityImpl) recheckCAA(ctx context.Context, rechecks []string) error {
 	ra.stats.Inc("recheck_caa", 1)
-	ra.stats.Inc("recheck_caa_names", int64(len(names)))
+	ra.stats.Inc("recheck_caa_names", int64(len(rechecks)))
 	wg := sync.WaitGroup{}
-	ch := make(chan *probs.ProblemDetails, len(names))
-	for _, name := range names {
+	ch := make(chan *probs.ProblemDetails, len(rechecks))
+	for _, name := range rechecks {
```

Comment:
Ping on this; it still has the "names -> rechecks" diff.

Reply:
Oops! Thanks for the ping, I forgot this one. Fixed in f3008c8